latest master fails to build on ubuntu 18.04.

(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: ubuntu 18.04 

**Describe the bug**
after running:

`cmake ..`

end up with:
```
[...]
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Checking for module 'portaudiocpp >= 12'
--   Found portaudiocpp , version 12
-- Checking for module 'sndfile >= 1.0.25'
--   Found sndfile , version 1.0.28
-- Checking for module 'gtk-mac-integration'
--   No package 'gtk-mac-integration' found
-- Checking for module 'lua5.3'
--   Found lua5.3, version 5.3.3
Enable Xournal++ Plugins
-- Found Gettext: /usr/bin/xgettext (found version "0.19.8.1") 
CMake Error at src/CMakeLists.txt:87 (target_link_libraries):
  Object library target "xournalpp-core" may not link to anything.

-- Configuring incomplete, errors occurred!
```

**To Reproduce**
following build steps: 
```
git clone http://github.com/xournalpp/xournalpp.git
cd xournalpp
mkdir build
cd build 
cmake ..    #fails here
cmake --build .
```

**Expected behavior**
build completes


**Additional context**
first noticed this a couple of weeks back.

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -84,7 +84,8 @@ unset (xournalpp_SOURCES_RECURSE)
 add_library (xournalpp-core OBJECT ${xournalpp_SOURCES})
 add_dependencies (xournalpp-core util)
 target_compile_features (xournalpp-core PUBLIC ${PROJECT_CXX_FEATURES})
-target_link_libraries(xournalpp-core util)
+# Can't use target_link_directories since we require CMake >= 3.10
+target_include_directories (xournalpp-core PRIVATE $<TARGET_PROPERTY:util,INTERFACE_INCLUDE_DIRECTORIES>)
 
 ## xournalpp main program ##
 add_executable (xournalpp

